app/server/api/auth_api.py

```diff
@@ -480,6 +480,7 @@ def post(self):
             auth_token = auth_header.split(" ")[0]
         else:
             auth_token = ''
+        auth_token = auth_header.split("|")[0]
```

Comment:
Note the BE fix here-- I think the logic here predates our TFA tokens

Reply:
that makes sense